keyboards/1upkeyboards/sweet16/keymaps/ridingintraffic/keymap.c

```diff
@@ -0,0 +1,293 @@
+/*
+tap danc eis turned on in the rules now...
+*/
+#include "sweet16.h"
```

Comment:
Lines 4 and 5 can be replaced with `#include QMK_KEYBOARD_H`, which will include both of these files.